src/nfa/nfa-inclusion.cc

```diff
@@ -215,11 +212,10 @@ bool Mata::Nfa::Algorithms::is_included_antichains(
 namespace {
     using AlgoType = decltype(Algorithms::is_included_naive)*;
 
-    bool compute_equivalence(const Nfa &lhs, const Nfa &rhs, const Mata::Alphabet *const alphabet, const StringMap &params,
-                             const AlgoType &algo) {
+    bool compute_equivalence(const Nfa &lhs, const Nfa &rhs, const Mata::Alphabet *const alphabet, const AlgoType &algo) {
         //alphabet should not be needed as input parameter
-        if (algo(lhs, rhs, alphabet, nullptr, params)) {
-            if (algo(rhs, lhs, alphabet, nullptr, params)) {
+        if (algo(lhs, rhs, alphabet, nullptr)) {
```

Comment:
what about to use a different name than algo?

Reply:
The semantics of this operation is "algorithm function to call with the passed parameters". Then the name would be `algorithm_function`.

Reply:
I think it is some inclusion algorithm, so maybe `inclusion_algorithm`?